Sources/MongoSwift/MongoCollection+BulkWrite.swift

```diff
@@ -53,12 +56,17 @@ extension MongoCollection {
         }
 
         /// Adds the `deleteOne` operation to a bulk write
+        ///
+        /// - Throws:
+        ///   - `UserError.invalidArgumentError` if the options form an invalid combination.
+        ///   - `EncodingError` if an error occurs while encoding the options to BSON.
         public func addToBulkWrite(bulk: BulkWriteOperation, index: Int) throws {
-            let opts = try BSONEncoder().encode(self.options)
+            let opts = try BSONEncoder().encode(options)
```

Comment:
I thought we _were_ doing explicit `self`?

Reply:
the swiftlint rule seems to miss a lot of these... 😩